tests/golos.referral_tests.cpp

```diff
@@ -138,8 +141,8 @@ BOOST_FIXTURE_TEST_CASE(transfer_tests, golos_referral_tester) try {
 
     BOOST_TEST_MESSAGE("--- issue tokens for users");
     BOOST_CHECK_EQUAL(success(), token.create(cfg::emission_name, token.make_asset(10000)));
-    BOOST_CHECK_EQUAL(success(), token.issue(cfg::emission_name, N(vania), token.make_asset(30), "issue 30 tokens for vania"));
-    BOOST_CHECK_EQUAL(success(), token.issue(cfg::emission_name, N(tania), token.make_asset(30), "issue 30 tokens for tania"));
+    BOOST_CHECK_EQUAL(success(), token.issue(cfg::emission_name, N(vania), token.make_asset(300), "issue 300 tokens for vania"));
```

Comment:
Value equal `30` in original code. Please, return this value to original state (if it does not affect the tests).